statemachine/aws-stepfunctions-statemachine.json

```diff
@@ -178,6 +178,11 @@
   "createOnlyProperties": [
     "/properties/StateMachineName"
   ],
+  "writeOnlyProperties": [
```

Comment:
Synced offline.
We might not supporting drift detection right now before this change because Read Handler is not returning all properties. 
We can proceed with this PR